.circleci/m2/settings.xml

```diff
@@ -0,0 +1,16 @@
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0
+  http://maven.apache.org/xsd/settings-1.0.0.xsd">
+    <servers>
+        <server>
+            <id>packagecloud-librato</id>
+            <password>${env.PACKAGECLOUD_TOKEN}</password>
```

Comment:
do we need packagecloud?

Reply:
Yeah, left-overs from copied circle build